CoffeeChat/Controller/AboutPearViewController.swift

```diff
@@ -0,0 +1,174 @@
+        setShadow(button: moreAppsButton)
+        view.addSubview(moreAppsButton)
+        
+        setupConstraints()
+        setupNavigationBar()
+    }
+    
+    private func setupNavigationBar() {
+        navigationController?.navigationBar.isHidden = false
+        navigationController?.navigationBar.barTintColor = .backgroundLightGreen
+        navigationController?.navigationBar.shadowImage = UIImage()
+        navigationController?.navigationBar.titleTextAttributes = [
+            .font: UIFont.getFont(.medium, size: 24)
+        ]
+
+        backButton.setImage(UIImage(named: "backArrow"), for: .normal)
+        backButton.snp.makeConstraints { (make) in
+            make.size.equalTo(CGSize(width: 10, height: 20))
+        }
+        backButton.addTarget(self, action: #selector(backPressed), for: .touchUpInside)
+        let backBarButtonItem = UIBarButtonItem(customView: backButton)
+        navigationItem.leftBarButtonItem = backBarButtonItem
+    }
+    
+    private func setShadow(button: UIButton) {
+        button.layer.shadowColor = UIColor(red: 0, green: 0, blue: 0, alpha: 0.15).cgColor
+        button.layer.shadowOffset = CGSize(width: 0, height: 2)
+        button.layer.shadowOpacity = 1
+        button.layer.shadowRadius = 4
+    }
+    
+    @objc private func backPressed() {
+        navigationController?.popViewController(animated: true)
+    }
+    
+    @objc private func sendFeedback() {
+        // Todo
+    }
+    
+    @objc private func visitWebsite() {
+        // Todo
+    }
+    
+    @objc private func presentMoreApps() {
+        // Todo
+    }
+    
+    private func setupConstraints() {
+        aboutLabel.snp.makeConstraints { (make) in
+            make.leading.trailing.equalToSuperview().inset(24)
+            make.top.equalTo(view.snp.top).offset(125)
+        }
+        aboutTableView.snp.makeConstraints { (make) in
+            make.top.equalTo(aboutLabel.snp.bottom).offset(40)
+            make.leading.trailing.equalToSuperview()
+            make.bottom.equalTo(view.snp.bottom).offset(-290)
+        }
+        feedbackButton.snp.makeConstraints { (make) in
+            make.centerX.equalTo(view.snp.centerX)
+            make.top.equalTo(aboutTableView.snp.bottom).offset(10)
+            make.width.equalTo(180)
+            make.height.equalTo(40)
+        }
+        visitWebsiteButton.snp.makeConstraints { (make) in
+            make.centerX.height.width.equalTo(feedbackButton)
+            make.top.equalTo(feedbackButton.snp.bottom).offset(12)
+        }
+        moreAppsButton.snp.makeConstraints { (make) in
+            make.centerX.width.height.equalTo(feedbackButton)
+            make.top.equalTo(visitWebsiteButton.snp.bottom).offset(12)
+        }
+    }
+
+}
+
+extension AboutPearViewController: UITableViewDataSource, UITableViewDelegate {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
```

Comment:
newline. Also, I personally separate each extension even if they're related, but I'd check to see what the Pear codebase does just to be consistent.